<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>removeStudentDialog</class>
 <widget class="QDialog" name="removeStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ok_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="studentNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Students</string>
   </property>
  </widget>
  <widget class="QListView" name="studentListView">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
